Update tools.rst

mwalls opened the pull request:
Trying to fix more links. Hope the ones I've done are ok. But there are some broken ones here I don't know how to fix. (data, original metadata, angle and others).
I'll try and find out unless someone else can see immediately...

mwalls committed:
Update tools.rst

Trying to fix more links. Hope the ones I've done are ok. But there are some broken ones here I don't know how to fix. (data, original metadata, angle and others).
I'll try and find out unless someone else can see immediately...

mwalls committed:
Update tools.rst

there was a mistake (at least one)

mwalls committed:
Update tools.rst

francisco-dlp commented on doc/user_guide/tools.rst:
```diff
@@ -58,12 +58,12 @@ examples, the data is stored in a numpy array in the
 in the :py:attr:`~.signal.BaseSignal.metadata` attribute and the axes
 information (including calibration) can be accessed (and modified) in the
-:py:attr:`~.signal.BaseSignal.axes_manager` attribute.
+:py:class:`~.axes.:py:class:`~.axes.AxesManager`` attribute.
```
``py:class:`~.axes.AxesManager``  instead?

francisco-dlp commented on doc/user_guide/tools.rst:
```diff
@@ -58,12 +58,12 @@ examples, the data is stored in a numpy array in the
 ---------------------
 
-Many of the values in the :py:attr:`~.signal.BaseSignal.axes_manager` can be
+Many of the values in the :py:class:`~.axes.:py:class:`~.axes.AxesManager`` can be
```
As above

mwalls replied:
Actally, maybe I somehow changed that last line myself? It seems I did. I'll do it again Sorry.

francisco-dlp reviewed (changes requested):
Thanks! Just a minor issue to fix.

mwalls committed:
Update tools.rst

Thanks. Is this what you mean? As you can tell I'm doing this by picking things that seem to  work elsewhere, without having the slightest knowledge of the syntax. 
I picked that bit of code form the last line of this file:
"Both are available via the :py:class:`~.axes.:py:class:`~.axes.AxesManager`` of the signal."
For which the link works on the site. My problem is not being able to test the code. Is that what "Sphinx" is about? Do I have to learn another thing? Arggh, I'm too old!!

mwalls committed:
Update tools.rst

OK now?
Sorry for the fuss

francisco-dlp commented:
Thanks!
